examples/src/app/helpers/raw-file-loading.ts

```diff
@@ -0,0 +1,114 @@
+import { File } from './types';
```

Comment:
some of the files have no comments at all .. it'd be good to add at least some description of the purpose of the file / class.